extensions/winrt/_app_service/AppServiceConnection.cpp

```diff
@@ -0,0 +1,239 @@
+#include "pch.h"
+#include "event.h"
+#include "Types.h"
+
+using namespace Microsoft::WRL;
+using namespace Windows::Foundation;
+using namespace Windows::ApplicationModel::AppService;
+
+namespace PyAppServiceConnection
+{
+    char ServiceClosedEventName[] = "ServiceClosed";
+    char RequestReceivedEventName[] = "RequestReceived";
+
+    static int init(PyObject *obj, PyObject *args, PyObject *kwds)
+    {
+        const int FailureValue = -1;
+        return ExceptionGuard([=]
+        {
+            PyWinRTObject *self = reinterpret_cast<PyWinRTObject*>(obj);
+            if (self->pInspectable)
+            {
+                self->pInspectable->Release();
+                self->pInspectable = NULL;
+            }
+
+            if (!self->dict)
+            {
+                self->dict = PyDict_New();
+            }
+
+            auto thisObj = ref new AppServiceConnection();
+
+            using namespace Windows::Foundation;
```

Comment:
No.  I moved it to the top of the file, but forgot to remove this one.  I'll remove it. Thanks.